app/domain/authentication/authn_jwt/restriction_validators/validate_restrictions_one_to_one.rb

```diff
@@ -9,14 +9,16 @@ def initialize(decoded_token:)
         end
 
         def valid_restriction?(restriction)
```

Comment:
there is any possibility that the restriction will be blank and we will get nil function exception ?

Reply:
Restrictions are created from annotations loaded in ExtractResourceRestrictions and verified by costaints so no.